kolibri/core/logger/csv_export.py

```diff
@@ -89,7 +95,7 @@ def map_object(obj):
 classes_info = {
     "session": {
         "queryset": ContentSessionLog.objects.all(),
-        "filename": "content_session_logs.csv",
+        "filename": "{}_content_session_logs.csv",
```

Comment:
Did you intend to use the `CSV_EXPORT_FILENAMES` dict to replace these values? e.g. `CSV_EXPORT_FILENAMES["session"]`

Reply:
Yep, sorry I didn't finish the refactoring. It's fixed  now